javascript/ql/src/Security/CWE-078/IndirectCommandInjection.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Indirect uncontrolled command line
+ * @description Using command-line arguments to construct other
+ *              command-line arguments may allow a malicious user to
+ *              indirectly change the meaning of the command.
```

Comment:
Again, the threat model needs to be explained more clearly. Based on your previous explanations I now understand what you mean by "indirectly" here, but someone reading this description for the first time might not.